### PR TITLE
[RHCLOUD-17965] Add Oracle Cloud Infraestructure as a new source

### PR DESCRIPTION
**Pull request by MikelAlejoBR**
Adds Oracle Cloud Infraestructure as a source, by supporting the [Oracle Cloud IDs (OCIDs)](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/identifiers.htm)

## Links

[[RHCLOUD-17965]](https://issues.redhat.com/browse/RHCLOUD-17965)

### CODE REVIEW

File: dao/seeds/source_types.yml [hunk 1]
```diff
@@ -217,6 +217,103 @@ azure:
...
+            hideField: true
+            initializeOnMount: true
+            initialValue: gitlab-personal-access-token
+          - component: text-field
+            name: authentication.username
+            label: Username of the GitLab account
+          - component: text-field
+            name: authentication.password
+            label: Personal Acces Token for the GitLab account
+oracle-cloud-infraestructure:
```
Comment (lindgrenj6) [inline dao/seeds/source_types.yml]: Hmmm. This is showing as `oracle-cloud-infraestructure` to me - is it supposed to be `oracle-cloud-infrastructure`?

**Review by lindgrenj6 — COMMENTED** [review]
Sorry for making this conflict, but I think I see a typo as well.

File: dao/seeds/source_types.yml (hunk 1 above)
Comment (MikelAlejoBR) [reply dao/seeds/source_types.yml]: Whoops! In Spanish, the word for "infrastructure" is "infraestructura", so it seems that I mixed both languages :joy:

**Review by lindgrenj6 — APPROVED** [review]

**Pull request merged**